astronomer/providers/microsoft/azure/example_dags/example_wasb_sensors.py

```diff
@@ -0,0 +1,157 @@
+import os
+from datetime import datetime, timedelta
+
+from airflow import DAG
+from airflow.operators.empty import EmptyOperator
+from airflow.operators.python import PythonOperator
+
+from astronomer.providers.microsoft.azure.sensors.wasb import (
+    WasbBlobSensorAsync,
+    WasbPrefixSensorAsync,
+)
+
+AZURE_DATA_STORAGE_BLOB_NAME = os.getenv("AZURE_DATA_STORAGE_BLOB_NAME", "test_blob_providers_team.txt")
+AZURE_DATA_STORAGE_CONTAINER_NAME = os.getenv(
+    "AZURE_DATA_STORAGE_CONTAINER_NAME", "test-container-providers-team"
+)
+AZURE_WASB_CONN_ID = os.getenv("AZURE_WASB_CONN_ID", "wasb_default")
+EXAMPLE_AZURE_TEST_FILE_PATH = "/usr/local/airflow/dags/example_azure_test_file.txt"
+EXECUTION_TIMEOUT = int(os.getenv("EXECUTION_TIMEOUT", 6))
+
+
+default_args = {
+    "execution_timeout": timedelta(hours=EXECUTION_TIMEOUT),
+    "retries": int(os.getenv("DEFAULT_TASK_RETRIES", 2)),
+    "retry_delay": timedelta(seconds=int(os.getenv("DEFAULT_RETRY_DELAY_SECONDS", 10))),
+}
+
+
+def create_azure_data_storage_container_callable(container_name: str) -> None:
+    """
+    Creates an Azure Data Storage container.
+
+    If a container with the given name already exists, it just logs saying that and does not raise an Exception.
+
+    :param container_name: name of the data storage container to be created
+    """
+    from airflow.providers.microsoft.azure.hooks.wasb import WasbHook
+
+    hook = WasbHook(wasb_conn_id=AZURE_WASB_CONN_ID)
+    hook.create_container(container_name)
+
+
+def delete_azure_data_storage_container_callable(container_name: str) -> None:
+    """
+    Deletes the Azure Data Storage container.
+
+    :param container_name: name of the data storage container to be deleted
+    """
+    from airflow.providers.microsoft.azure.hooks.wasb import WasbHook
+
+    hook = WasbHook(wasb_conn_id=AZURE_WASB_CONN_ID)
+    hook.delete_container(container_name)
+
+
+def upload_blob_callable(file_path: str, container_name: str, blob_name: str) -> None:
+    """
+    Uploads the given example file as a blob in the given container.
+
+    It raises an exception if the specified name of the blob is less that 10 characters with an interest to test
```

Comment:
It raises an exception if the specified name of the blob is less than 10 characters with an interest to test